builder/parser/line_parsers.go

```diff
@@ -209,34 +208,28 @@ func parseString(rest string) (*Node, map[string]bool, error) {
 
 // parseJSON converts JSON arrays to an AST.
 func parseJSON(rest string) (*Node, map[string]bool, error) {
-	var (
-		myJson   []interface{}
-		next     = &Node{}
-		orignext = next
-		prevnode = next
-	)
-
+	var myJson []interface{}
```

Comment:
(because if something fails to parse here, it actually just gets passed to the shell instead, unless it's this one specific special error)